### ISSUE TITLE
Exception with sbt-web 1.4.0

### ISSUE THREAD

**Issue by tptodorov** [sbt/sbt-uglify#25]
Using the plugin with sbt-web 1.4.0 causes the following:
```

java.lang.NoSuchMethodError: com.typesafe.sbt.web.SbtWeb$.syncMappings(Ljava/io/File;Lscala/collection/Seq;Ljava/io/File;)Ljava/io/File;
	at com.typesafe.sbt.uglify.SbtUglify$$anonfun$runOptimizer$1$$anonfun$apply$3.apply(SbtUglify.scala:92)
	at com.typesafe.sbt.uglify.SbtUglify$$anonfun$runOptimizer$1$$anonfun$apply$3.apply(SbtUglify.scala:86)
	at scala.Function$$anonfun$chain$1$$anonfun$apply$1.apply(Function.scala:24)
	at scala.Function$$anonfun$chain$1$$anonfun$apply$1.apply(Function.scala:24)
	at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:111)
	at scala.collection.immutable.List.foldLeft(List.scala:84)
	at scala.collection.TraversableOnce$class.$div$colon(TraversableOnce.scala:138)
	at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:105)
	at scala.Function$$anonfun$chain$1.apply(Function.scala:24)
	at com.typesafe.sbt.web.SbtWeb$$anonfun$projectSettings$44.apply(SbtWeb.scala:226)
	at com.typesafe.sbt.web.SbtWeb$$anonfun$projectSettings$44.apply(SbtWeb.scala:226)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] (*:webPipeline) java.lang.NoSuchMethodError: com.typesafe.sbt.web.SbtWeb$.syncMappings(Ljava/io/File;Lscala/collection/Seq;Ljava/io/File;)Ljava/io/File;
```

**Comment by danielnixon** [comment]
Fixed in [my fork](https://github.com/danielnixon/sbt-uglify). Try `addSbtPlugin("org.danielnixon" % "sbt-uglify" % "1.0.7")`.

**Comment by tptodorov** [comment]
thanks! this works now.

**Comment by ldaume** [comment]
thanks @danielnixon. Why isn't it merged into this repo?

**Comment by ndimas** [comment]
I agree, I have this error on my Windows machine with sbt-web 1.4.1 too

**Comment by jroper** [comment]
This should be fixed in sbt uglify 2.0.0 (which now built with and tested against sbt-web 1.4.2, and is compatible with sbt 1.0).
